### PR TITLE
Updated documentation for AC

### PR DESCRIPTION
**Pull request by Anna-Mironova**
Added missing descriptions, fixed typos

### CODE REVIEW

File: tools/accuracy_checker/accuracy_checker/launcher/dlsdk_launcher_readme.md [hunk 1]
```diff
@@ -1,42 +1,23 @@
 # How to configure OpenVINO™ launcher
 
+OpenVINO™ launcher is one of the supported wrappers for easily launching models within Accuracy Checker tool. This launcher allows to execute models in the Intermediate Representation (IR) format.
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/launcher/dlsdk_launcher_readme.md:3]: not only IR
```suggestion
OpenVINO™ launcher is one of the supported wrappers for easily launching models within Accuracy Checker tool. This launcher uses OpenVINO™ Inference Engine as inference backend and accept for executing networks in OpenVINO™ supported formats.
```

File: tools/accuracy_checker/accuracy_checker/adapters/README.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
 # Adapters
 
-Adapter is a function for conversion network infer output to metric specific format.
+Adapter is a function for converting network infer output to metric specific format. Adapters may have parameters available for configuration. The adapter and its parameters, if necessary, are set through the configuration file.
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/adapters/README.md:3]:
```suggestion
Adapter is a class for converting raw network infer output to metric specific format. Adapters may have parameters available for configuration. The adapter and its parameters, if necessary, are set through the configuration file.
```

File: tools/accuracy_checker/accuracy_checker/data_readers/README.md [hunk 3]
```diff
@@ -1,14 +1,19 @@
 # Data Readers
 
-Data Reader is a function for reading input data.
-You can use 2 ways to set data_reader for dataset:
+Data Reader is a function for reading input data in a specific format and converting it to a format suitable for model inference. Readers may have parameters available for configuration. The reader and its parameters, if necessary, are set through the configuration file.
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/data_readers/README.md:3]: I think data conversion is extra info, which cam mislead users
```suggestion
Data Reader is a class for reading input data in a specific format. Readers may have parameters available for configuration. The reader and its parameters, if necessary, are set through the configuration file.
```

File: tools/accuracy_checker/accuracy_checker/launcher/dlsdk_launcher_readme.md [hunk 4]
```diff
@@ -1,42 +1,23 @@
...
 * `model` - path to xml file with model for your topology or compiled executable network.
 * `weights` - path to bin file with weights for your topology (Optional, the argument can be omitted if bin file stored in the same directory with model xml or if you use compiled blob).
+* `adapter` - approach how raw output will be converted to representation of dataset problem, some adapters can be specific to framework. You can find detailed instruction how to use adapters [here](../adapters/README.md).
 
 **Note:**
    You can generate executable blob using [compile_tool](https://docs.openvinotoolkit.org/latest/_inference_engine_tools_compile_tool_README.html).
    Before evaluation executable blob, please make sure that selected device support it.
 
-
-launcher may optionally provide model parameters in source framework format which will be converted to Inference Engine IR using Model Optimizer.
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/launcher/dlsdk_launcher_readme.md:17]: why running with using MO conversion part is removed?
We still support this functionaity

File: tools/accuracy_checker/README.md [hunk 5]
```diff
@@ -1,5 +1,7 @@
 # Deep Learning accuracy validation framework
 
+The Accuracy Checker is an extensible, flexible and configurable Deep Learning accuracy validation framework. The tool has a modular structure and allows to reproduce validation pipeline and collect aggregated quality indicators for popular datasets both for networks in source frameworks and in the Intermediate Representation (IR) format.
```
Comment (eaidova) [inline tools/accuracy_checker/README.md:3]:
```suggestion
The Accuracy Checker is an extensible, flexible and configurable Deep Learning accuracy validation framework. The tool has a modular structure and allows to reproduce validation pipeline and collect aggregated quality indicators for popular datasets both for networks in source frameworks and in the OpenVINO.
```

File: tools/accuracy_checker/accuracy_checker/launcher/onnx_runtime_launcher_readme.md [hunk 6]
```diff
@@ -1,5 +1,7 @@
 # How to configure ONNX Runtime launcher
 
+ONNX Runtime launcher is one of the supported wrappers for easily launching models within Accuracy Checker tool. This launcher allows to execute models in ONNX framework.
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/launcher/onnx_runtime_launcher_readme.md:3]:
```suggestion
ONNX Runtime launcher is one of the supported wrappers for easily launching models within Accuracy Checker tool. This launcher allows to execute models in ONNX format using ONNX Runtime as inference backend.
```

File: tools/accuracy_checker/accuracy_checker/metrics/README.md [hunk 7]
```diff
@@ -1,5 +1,6 @@
 # Metrics
 
+Metric is a function which is used to evaluate and compare predicted data with annotated data.
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/metrics/README.md:3]:
```suggestion
Metric is a class which is used to compare predicted data with annotated data and perform quality measurement.
```

File: tools/accuracy_checker/accuracy_checker/postprocessor/README.md [hunk 8]
```diff
@@ -1,13 +1,15 @@
 # Postprocessors
 
-Postprocessor is function which processes prediction and/or annotation data after model infer and before metric calculation. For correct work postprocessors require specific representation format.
+Postprocessor is function which processes prediction and/or annotation data after model inference and before metric calculation. For correct work postprocessors require specific representation format.
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/postprocessor/README.md:3]:
```suggestion
Postprocessor is class which processes prediction and/or annotation data after model inference and before metric calculation. For correct work postprocessors require specific representation format.
```

File: tools/accuracy_checker/accuracy_checker/preprocessor/README.md [hunk 9]
```diff
@@ -1,7 +1,9 @@
 # Preprocessors
 
-Preprocessor is function which processes input data before model inference.
-Every preprocessor has parameters available for configuration.
+Preprocessor is function which processes input data before model inference. Every preprocessor has parameters available for configuration. The preprocessor and its parameters are set through the configuration file. Preprocessors are provided in `datasets` section of configuration file to use specific preprocessor.
```
Comment (eaidova) [inline tools/accuracy_checker/accuracy_checker/preprocessor/README.md:3]:
```suggestion
Preprocessor is class which processes input data before model inference. Every preprocessor has parameters available for configuration. The preprocessor and its parameters are set through the configuration file. Preprocessors are provided in `datasets` section of configuration file to use specific preprocessor.
```

File: tools/accuracy_checker/accuracy_checker/postprocessor/README.md (hunk 8 above)
Comment (eaidova) [reply tools/accuracy_checker/accuracy_checker/postprocessor/README.md:3]: possibly information about what the difference between adapter and postprocessor will be useful

File: tools/accuracy_checker/accuracy_checker/launcher/dlsdk_launcher_readme.md (hunk 1 above)
Comment (Anna-Mironova) [reply tools/accuracy_checker/accuracy_checker/launcher/dlsdk_launcher_readme.md]: Done

File: tools/accuracy_checker/accuracy_checker/adapters/README.md (hunk 2 above)
Comment (Anna-Mironova) [reply tools/accuracy_checker/accuracy_checker/adapters/README.md]: Done

File: tools/accuracy_checker/accuracy_checker/data_readers/README.md (hunk 3 above)
Comment (Anna-Mironova) [reply tools/accuracy_checker/accuracy_checker/data_readers/README.md]: Done

File: tools/accuracy_checker/accuracy_checker/launcher/caffe_launcher_readme.md [hunk 10]
```diff
@@ -1,5 +1,7 @@
 # How to configure Caffe launcher
 
+Caffe launcher is one of the supported wrappers for easily launching models within Accuracy Checker tool. This launcher allows to execute models in Caffe framework.
```
[the remark this answers is not in the document]
Comment (Anna-Mironova) [reply tools/accuracy_checker/accuracy_checker/launcher/caffe_launcher_readme.md]: Done

File: tools/accuracy_checker/README.md (hunk 5 above)
Comment (Anna-Mironova) [reply tools/accuracy_checker/README.md]: Done

File: tools/accuracy_checker/accuracy_checker/launcher/onnx_runtime_launcher_readme.md (hunk 6 above)
Comment (Anna-Mironova) [reply tools/accuracy_checker/accuracy_checker/launcher/onnx_runtime_launcher_readme.md]: Done

File: tools/accuracy_checker/accuracy_checker/metrics/README.md (hunk 7 above)
Comment (Anna-Mironova) [reply tools/accuracy_checker/accuracy_checker/metrics/README.md]: Done

File: tools/accuracy_checker/accuracy_checker/postprocessor/README.md (hunk 8 above)
Comment (Anna-Mironova) [reply tools/accuracy_checker/accuracy_checker/postprocessor/README.md]: Done

File: tools/accuracy_checker/accuracy_checker/preprocessor/README.md (hunk 9 above)
Comment (Anna-Mironova) [reply tools/accuracy_checker/accuracy_checker/preprocessor/README.md]: Done

**Review by eaidova — APPROVED** [review]

**Pull request merged**